Source/Eto.Gtk/Forms/Controls/FontPickerHandler.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using Eto.Drawing;
+using Eto.Forms;
+using Eto.GtkSharp.Drawing;
+
+namespace Eto.GtkSharp.Forms.Controls
+{
+	public class FontPickerHandler : GtkControl<Gtk.FontButton, FontPicker, FontPicker.ICallback>, FontPicker.IHandler
+	{
+		public Font Value
+		{
+			get { return new Font(new FontHandler(Control.FontName)); }
+			set
+			{
+				if (value == null)
+					Control.FontName = string.Empty;
+				else
+				{
+					var handler = value.Handler as FontHandler;
+					Control.FontName = handler.Control.ToString();
+				}
+			}
+		}
+
+		public FontPickerHandler()
+		{
+			Control = new Gtk.FontButton();
+		}
+
+		public override void AttachEvent(string id)
+		{
+			switch (id)
+			{
+				case FontPicker.ValueChangedEvent:
+					Control.FontSet += (sender, e) => Callback.OnValueChanged(Widget, EventArgs.Empty);
```

Comment:
You might need to fix up some stuff: https://github.com/picoe/Eto/blob/develop/Source/Eto/Forms/ThemedControls/ThemedTextStepperHandler.cs#L228